cmd/yggd/grpc.go

```diff
@@ -46,6 +47,10 @@ func newDispatcher(httpClient *http.Client) *dispatcher {
 	}
 }
 
+func (d *dispatcher) Reload(client *http.Client) {
+	d.httpClient = client
```

Comment:
Would it make more sense to just make the `httpClient` struct field public and set the field directly? "Reload" is a bit of a misnomer here. It doesn't actually reload anything; it just sets a struct field value.